modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -467,15 +467,17 @@ const EditFDSFieldModalContent = ({
 	};
 
 	const validateFDSField = function () {
-		const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+		if (Liferay.FeatureFlags['LPS-172017']) {
+			const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
 
-		if (!i18nFieldLabels[defaultLanguageId]) {
-			setErrorMessage(Liferay.Language.get('required'));
+			if (!i18nFieldLabels[defaultLanguageId]) {
+				setErrorMessage(Liferay.Language.get('required'));
```

Comment:
Outside of scope of this task, but we should remove required from fields as well.